src/proxy/reconnect.rs

```diff
@@ -46,7 +49,7 @@ where
 #[derive(Clone, Debug)]
 enum Backoff {
     None,
-    Fixed(Duration),
+    Exponential(Duration, Duration, f64),
```

Comment:
I think it would be a little clearer to use named params here, especially if we make this struct `pub`, e.g.:
```rust
pub enum Backoff {
    None,
    Exponential { min: Duration, max: Duration, jitter: f64 },
}
```